examples/retinanet/model.py

```diff
@@ -0,0 +1,412 @@
+from anchor import *
+from util import *
+from flax.nn.initializers import normal
+from jax import numpy as jnp
+
+import flax
+import jax
+
+
+class ClassificationSubnet(flax.nn.Module):
```

Comment:
Did you consider switching to Linen?
https://colab.sandbox.google.com/drive/1KPkQ_Hee9HlveynIF0FgPBzZDK79uoWL

Reply:
Yes, if Dan has time for doing the switch he will do it. However Linen examples live in a separate folder and we should first submit this example before forking it into the linen_examples folder.

Reply:
Absolutely. I will try to address this in a PR in the following week or so.